spec/support/change_streams.rb

```diff
@@ -156,9 +156,9 @@ def run
         end
 
         def teardown_test
-          @global_client && @global_client.close
           @db1 && @db1.drop
           @db2 && @db2.drop
+          @global_client && @global_client.close
```

Comment:
Dropping the databases raised an error after `close_stale_sockets!` was changed not to reconnect the sockets, so this needed to be updated. Given that `@db1` and `@db2` were created from `@global_client`, I think this is more correct anyways.